webapp/src/containers/Rhs/rhs.scss

```diff
@@ -48,8 +48,11 @@
 }
 
 .rhs-btn-container {
-  @include positioning(fixed, "", 0, 0, 1px, 1);
-  padding-left: 11px;
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  left: 1px;
+  z-index: 1;
```

Comment:
Have you checked if this styling breaks the UI on high server versions?

Reply:
yes, checked on the mm demo (v6.7.2)